React/Views/RCTMapManager.m

```diff
@@ -73,6 +74,27 @@ - (void)mapView:(RCTMap *)mapView didSelectAnnotationView:(MKAnnotationView *)vi
   }
 }
 
+- (void) mapView:(RCTMap *)mapView annotationView:(MKAnnotationView *)view didChangeDragState:(MKAnnotationViewDragState)newState fromOldState:(MKAnnotationViewDragState)oldState
+{
+  if ([view.annotation isKindOfClass:[RCTPointAnnotation class]]) {
+    
+    RCTPointAnnotation *annotation = (RCTPointAnnotation *)view.annotation;
+    
+    if (mapView.onAnnotationDragChange) {
+      mapView.onAnnotationDragChange(@{
+        @"state": @(newState),
```

Comment:
Is this state property useful? If it is, we should probably convert it to a more JS-friendly string constant, and define an enum for it in the js module.